.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "moon-cow-simple/entry-blue-cow-zero"]
 	path = moon-cow-simple/entry-blue-cow-zero
 	url = https://github.com/nicholassmith/blue-cow-zero
+[submodule "moon-cow-simple/entry-jimallanson-gpt2"]
```

Comment:
will this mean I have to download 139GB of model data?

Reply:
Only ~4GB I think.